extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -623,74 +659,73 @@ export class AmpLightboxViewer extends AMP.BaseElement {
    */
   // TODO (cathyxz): make this generalizable to more than just images
   enter_(sourceImage) {
-    st.setStyles(this.element, {
-      opacity: 0,
-      display: '',
-    });
-
     const anim = new Animation(this.element);
     const dur = 500;
-
-    // Lightbox background fades in.
-    anim.add(0, tr.setStyles(this.element, {
-      opacity: tr.numeric(0, 1),
-    }), 0.6, ENTER_CURVE_);
-
-    // Try to transition from the source image.
     let transLayer = null;
-    if (sourceImage && isLoaded(sourceImage)) {
-      transLayer = this.element.ownerDocument.createElement('div');
-      transLayer.classList.add('i-amphtml-lightbox-viewer-trans');
-      this.element.ownerDocument.body.appendChild(transLayer);
-
-      const rect = layoutRectFromDomRect(sourceImage
-          ./*OK*/getBoundingClientRect());
-      const imageBox = this.elementsMetadata_[this.currentElemId_]
-          .imageViewer.getImageBox();
-
-      const clone = sourceImage.cloneNode(true);
-      clone.className = '';
-      st.setStyles(clone, {
-        position: 'absolute',
-        top: st.px(rect.top),
-        left: st.px(rect.left),
-        width: st.px(rect.width),
-        height: st.px(rect.height),
-        transformOrigin: 'top left',
-        willChange: 'transform',
-      });
-      transLayer.appendChild(clone);
-
-      sourceImage.classList.add('i-amphtml-ghost');
-
-      // Move and resize the image to the location given by the lightbox.
-      const dx = imageBox.left - rect.left;
-      const dy = imageBox.top - rect.top;
-      const scaleX = rect.width != 0 ? imageBox.width / rect.width : 1;
-
-      // Duration will be somewhere between 0.2 and 0.8 depending on how far
-      // the image needs to move.
-      const motionTime = Math.max(0.2, Math.min(0.8, Math.abs(dy) / 250 * 0.8));
-      anim.add(0, tr.setStyles(clone, {
-        transform: tr.concat([
-          tr.translate(tr.numeric(0, dx), tr.numeric(0, dy)),
-          tr.scale(tr.numeric(1, scaleX)),
-        ]),
-      }), motionTime, ENTER_CURVE_);
-
-      // At the end, fade out the transition image.
-      anim.add(0.9, tr.setStyles(transLayer, {
-        opacity: tr.numeric(1, 0.01),
-      }), 0.1, EXIT_CURVE_);
-    }
-
-    return anim.start(dur).thenAlways(() => {
-      sourceImage.classList.remove('i-amphtml-ghost');
-      st.setStyles(this.element, {opacity: ''});
-      st.setStyles(dev().assertElement(this.carousel_), {opacity: ''});
-      if (transLayer) {
-        this.element.ownerDocument.body.removeChild(transLayer);
+    return this.vsync_.measurePromise(() => {
```

Comment:
This long block of code change is a bit misleading, but the only thing changed here was to shove the measurements in a `vsync_.measurePromise`.